test/test_pull433.m

```diff
@@ -1,10 +1,11 @@
-function test_ft_electroderealign_headshape
```

Comment:
hmm, this commit should not have been here in this PR. But I think that the commit is fine in itself, so I'll leave it in.